NRP PS commandlet changes as per NRP Brooklyn APIs changes

Nilambari opened the pull request:
Will send another PR in some time with recorded tests. Can you Please approve this PR as without this merge its blocking for other team members to send their changes' PR. Thanks!

azurecla commented:
Hi __@Nilambari__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (nilamd). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Nilambari commented:
Added recorded tests back.

hovsepm commented on src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1:
```diff
@@ -80,17 +82,18 @@ function Test-VirtualNetworkGatewayConnectionCRUD
+      # Set/Update VirtualNetworkGatewayConnection
+	  $expected.Location = $location
+	  $expected.VirtualNetworkGateway1.Location = $location
+	  $expected.LocalNetworkGateway2.Location = $location
```
fix indentation.

Nilambari replied:
done,

hovsepm commented on src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/VirtualNetworkGatewayConnectionTests.ps1:
```diff
@@ -142,12 +145,12 @@ function Test-VirtualNetworkGatewayConnectionSharedKeyCRUD
       $vnetIpConfig = New-AzureVirtualNetworkGatewayIpConfig -Name $vnetGatewayConfigName -PublicIpAddress $publicip -Subnet $subnet
 
-      $actual = New-AzureVirtualNetworkGateway -ResourceGroupName $rgname -name $rname -location $location -IpConfigurations $vnetIpConfig -GatewaySize Default -GatewayType DynamicRouting -EnableBgp $false
+	  $actual = New-AzureVirtualNetworkGateway -ResourceGroupName $rgname -name $rname -location $location -IpConfigurations $vnetIpConfig -GatewayType Vpn -VpnType RouteBased -EnableBgp $false
```
indentation.

Nilambari replied:
done.

hovsepm commented:
Please include changes to changelog.txt file in your PR.

hovsepm commented:
Please fix indentation in several files in this PR. too many places to mention them one by one.

Nilambari commented:
Not sure about changlog.txt file. sharing overall changes in this PR:-
PS command changes as per NRP Brooklyn APIs changes:-
1. Command New-AzureLocalNetworkGateway parameter name change
2. Command Reset-AzureLocalNetworkGateway renamed to Set-AzureLocalNetworkGateway, added new parameter
3. Resource: VirtualNetworkGateway parameter changes
4. Command New-AzureVirtualNetworkGateway parameter changes
5. Removed command Resize-AzureVirtualNetworkGateway
6. Command Reset-AzureVirtualNetworkGatewayConnection renamed to Set-AzureVirtualNetworkGatewayConnection

Nilambari commented:
Added PS changes details in changeLog.txt file. Can you Please accept this PR? Thanks a lot!
